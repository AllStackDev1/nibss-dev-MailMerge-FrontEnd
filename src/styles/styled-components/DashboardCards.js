import styled from 'styled-components'

const Cards = styled.div`
                        margin-top: 30px;
                        &>div {
                            width: calc((100% - 45px) / 4);
                            height: 130px;
                            background: #FFF;
                            border-radius: 10px;
                        }
                    `;

const Card = styled.div`
                        &:first-of-type>div>div:first-of-type {
                            background: rgba(251, 185, 0, 0.3);
                        }
                        &:nth-of-type(2)>div>div:first-of-type {
                            background: rgba(93, 38, 132, 0.3);
                        }
                        &:nth-of-type(3)>div>div:first-of-type {
                            background: rgba(67, 207, 89, 0.3);
                        }
                        &:nth-of-type(4)>div>div:first-of-type {
                            background: rgba(51, 110, 214, 0.3);
                        }
                    `;

export { Cards, Card }
