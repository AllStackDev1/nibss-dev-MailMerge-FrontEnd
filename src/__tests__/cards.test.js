import React from 'react';
import { shallow } from 'enzyme';
import DashboardCards from "../components/Dashboard/snippets/dashboard/DashboardCards"



const shallowSetup = (props = {}) => {

    return shallow(<DashboardCards {...props} />)
}

it("should test conditions", () => {
    const wrapper = shallowSetup({ documents: { documents: {}, fetching: false } })
    expect(wrapper).toMatchSnapshot()
})

it("should test conditions", () => {
    const wrapper = shallowSetup({ documents: { documents: {}, fetching: true } })
    expect(wrapper).toMatchSnapshot()
})