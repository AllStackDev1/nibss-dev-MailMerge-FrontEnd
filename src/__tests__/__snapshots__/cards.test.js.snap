// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test conditions 1`] = `ShallowWrapper {}`;

exports[`should test conditions 2`] = `ShallowWrapper {}`;
